melodic/distribution.yaml

```diff
@@ -15510,6 +15510,24 @@ repositories:
       url: https://github.com/ElettraSciComp/witmotion_IMU_ros.git
       version: main
     status: maintained
+  witmotion_ros-release:
+    doc:
+      type: git
+      url: https://github.com/ElettraSciComp/witmotion_IMU_ros.git
```

Comment:
It is somewhat weird that the repository name is `witmotion_IMU_ros`, while the package name is `witmotion_ros`.  It is not a hard requirement to change it, but it is something to consider (github will automatically redirect in case you decide to do this).

Reply:
Asked for changing